apps/api/src/app/auth/services/auth.service.ts

```diff
@@ -239,4 +239,12 @@ export class AuthService {
   async verifyJwt(jwt: string) {
     return this.jwtService.verify(jwt);
   }
+
+  async validateRootEnvironment(payload: IJwtPayload) {
+    const environment = await this.environmentRepository.findOne({
+      _id: payload.environmentId,
+    });
+
+    return environment._parentId;
```

Comment:
```suggestion
    return !!environment._parentId;
```

A small suggestion on naming, since it's only returning this and not really performs any sort of validation there.